modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteAbstractDynamicCacheStartFailTest.java

```diff
@@ -772,4 +934,254 @@ public BrokenStoreFactory(boolean eOnAllNodes, String gridName) {
+        }
+
+        /** {@inheritDoc} */
+        @Override public ObjectInstance createMBean(String className, ObjectName name)
+            throws ReflectionException, InstanceAlreadyExistsException, MBeanRegistrationException,
+            MBeanException, NotCompliantMBeanException {
+            return origin.createMBean(className, name);
+        }
+
+        /** {@inheritDoc} */
+        @Override public ObjectInstance createMBean(String className, ObjectName name, ObjectName loaderName)
+            throws ReflectionException, InstanceAlreadyExistsException, MBeanRegistrationException,
+            MBeanException, NotCompliantMBeanException, InstanceNotFoundException {
+            return origin.createMBean(className, name, loaderName);
+        }
+
+        /** {@inheritDoc} */
+        @Override public ObjectInstance createMBean(String className, ObjectName name, Object[] params, String[] signature)
+            throws ReflectionException, InstanceAlreadyExistsException, MBeanRegistrationException,
+            MBeanException, NotCompliantMBeanException {
+            return origin.createMBean(className, name, params, signature);
+        }
+
+        /** {@inheritDoc} */
+        @Override public ObjectInstance createMBean(String className, ObjectName name,
+            ObjectName loaderName, Object[] params,
+            String[] signature) throws ReflectionException, InstanceAlreadyExistsException, MBeanRegistrationException,
+            MBeanException, NotCompliantMBeanException, InstanceNotFoundException {
+            return origin.createMBean(className, name, loaderName, params, signature);
+        }
+
+        /** {@inheritDoc} */
+        @Override public void unregisterMBean(ObjectName name) throws InstanceNotFoundException, MBeanRegistrationException {
+            origin.unregisterMBean(name);
+        }
+
+        /** {@inheritDoc} */
+        @Override public ObjectInstance getObjectInstance(ObjectName name) throws InstanceNotFoundException {
+            return origin.getObjectInstance(name);
+        }
+
+        /** {@inheritDoc} */
+        @Override public Set<ObjectInstance> queryMBeans(ObjectName name,
+            QueryExp query) {
+            return origin.queryMBeans(name, query);
+        }
+
+        /** {@inheritDoc} */
+        @Override public Set<ObjectName> queryNames(ObjectName name,
+            QueryExp query) {
+            return origin.queryNames(name, query);
+        }
+
+        /** {@inheritDoc} */
+        @Override public boolean isRegistered(ObjectName name) {
+            return origin.isRegistered(name);
+        }
+
+        /** {@inheritDoc} */
+        @Override public Integer getMBeanCount() {
+            return origin.getMBeanCount();
+        }
+
+        /** {@inheritDoc} */
+        @Override public Object getAttribute(ObjectName name,
+            String attribute) throws MBeanException, AttributeNotFoundException, InstanceNotFoundException, ReflectionException {

```

Comment:
Too long line. Move second argument to the previous line.

Reply:
done